src/test/java/org/junitpioneer/jupiter/EnvironmentVariableExtensionTests.java

```diff
@@ -255,17 +255,21 @@ void resetWarning() {
 		}
 
 		@Test
-		void shouldNotReportWarningIfExtensionNotUsed() {
+		@StdIo
+		void shouldNotReportWarningIfExtensionNotUsed(StdOut out) {
 			ExecutionResults results = executeTestMethod(ReportWarningTestCases.class, "testWithoutExtension");
 
 			assertThat(results).hasNoReportEntries();
+			assertThat((out.capturedLines())).containsExactly("");
```

Comment:
Why the extra parentheses?

Reply:
Er... _checks list of excuses_ ... readability?